### PR TITLE
Creation of awesome kw

### PR DESCRIPTION
**Pull request by OJarrisonn**
As talked in #1146, this creates the _awesome kw_ page for listing interresting resources for kworkflow

### CODE REVIEW

File: documentation/content/awesome_kw.rst
Comment (rodrigosiqueira) [inline documentation/content/awesome_kw.rst:1]: You need to add your new page to the index list; see the below diff:

```diff
diff --git a/documentation/index.rst b/documentation/index.rst
index a0e0ecf..4219792 100644
--- a/documentation/index.rst
+++ b/documentation/index.rst
@@ -25,6 +25,7 @@ check:
    content/codingstyle
    content/developmentworkflow
    content/tracingandprofiling
+   content/awesome_kw
 
 Kw code doc
 ===========
```

File: documentation/content/awesome_kw.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+===================
+ Awesome Kworkflow
+===================
```
Comment (rodrigosiqueira) [inline documentation/content/awesome_kw.rst:3]: Keep the `===...` under the title size.

Maybe rename the title to 'Awesome kworkflow related projects' or something similar.

File: documentation/content/awesome_kw.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Shell Completions
+-----------------
+
+bash
+____
+
+Completions for bash are installed by default when you install Kworkflow. The script can 
+be found at bash_autocomplete.sh_.
+
+.. _bash_autocomplete.sh: https://github.com/kworkflow/kworkflow/blob/unstable/src/bash_autocomplete.sh
```
Comment (rodrigosiqueira) [inline documentation/content/awesome_kw.rst:19]: This is internal to kw, you don't need to add it.

File: documentation/content/awesome_kw.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+nushell
+_______
```
Comment (rodrigosiqueira) [inline documentation/content/awesome_kw.rst:22]: Use `----` instead of `____`.

File: documentation/content/awesome_kw.rst [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+bash
+____
+
+Completions for bash are installed by default when you install Kworkflow. The script can 
+be found at bash_autocomplete.sh_.
+
+.. _bash_autocomplete.sh: https://github.com/kworkflow/kworkflow/blob/unstable/src/bash_autocomplete.sh
+
+nushell
```
Comment (rodrigosiqueira) [inline documentation/content/awesome_kw.rst:21]: Maybe use `Nushell`?

File: documentation/content/awesome_kw.rst [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+_______
+
+Nushell completions can be found at nu_scripts_.
+
+.. _nu_scripts: https://github.com/nushell/nu_scripts/tree/main/custom-completions/kw
+
+zsh
+___
+
+Zsh completions are also installed by default by Kworkflow.
```
Comment (rodrigosiqueira) [inline documentation/content/awesome_kw.rst:31]: You don't need to point to zsh internal file.

**Review by rodrigosiqueira — CHANGES_REQUESTED** [review]

**Comment by rodrigosiqueira** [comment]
@OJarrisonn any update here?

**Comment by OJarrisonn** [comment]
Sorry, i've been busy for the last couple days. But i'll make your suggested modifications asap

File: documentation/content/awesome_kw.rst (hunk 1 above)
Comment (OJarrisonn) [reply documentation/content/awesome_kw.rst:3]: The term _Awesome Y_  is a well known term that describes a collection of _awesome_ tools for _Y_.

See: [awesome-rust](https://github.com/rust-unofficial/awesome-rust), [awesome-go](https://github.com/avelino/awesome-go), [awesome-nu](https://github.com/nushell/awesome-nu)

**Comment by rodrigosiqueira** [comment]
I made some modifications, and after that, I merged your changes into the unstable branch. Thanks

**Pull request closed without merging**